### PR TITLE
CRBN-334: PoolApiController, BalanceChange removal from Postgres

### PR DESCRIPTION
**Pull request by jburley-nlok**

### CODE REVIEW

File: src/Miningcore/Api/Controllers/PoolApiController.cs [hunk 1]
```diff
@@ -448,49 +447,6 @@ public async Task<GetPoolResponse> GetPoolInfoAsync(string poolId)
         return response;
     }
 
-    [HttpGet("{poolId}/miners/{address}/balancechanges")]
```
Comment (kabilanvk) [inline src/Miningcore/Api/Controllers/PoolApiController.cs]: Can we keep these APIs and use cosmos to return data?

Comment (joel-nlok) [reply src/Miningcore/Api/Controllers/PoolApiController.cs]: we removed these APIs awhile back, when we deleted that table.  that is a good idea tho.  let's add a new story.

File: src/Miningcore/Extensions/DictionaryExtensions.cs [hunk 2]
```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2017 Coin Foundry (coinfoundry.org)
+Authors: Oliver Weichhold (oliver@weichhold.com)
```
Comment (joel-nlok) [inline src/Miningcore/Extensions/DictionaryExtensions.cs]: this is a wierd one.  we're adding a file (with OliverW's Copyright)  that wasn't in Oliver's repo?

**Review by joel-nlok — DISMISSED** [review]

**Comment by joel-nlok** [comment]
i approved it,  with just a couple questions to think about.

**Review by kabilanvk — APPROVED** [review]

**Review by joel-nlok — APPROVED** [review]

**Pull request merged**